content/files/Akeneo PIM API.postman_collection.json

```diff
@@ -2869,6 +2869,206 @@
                 "rest",
                 "v1",
                 "products"
```

Comment:
We should also add the following query parameters on GET /api/rest/v1/products/{code} :
- with_attribute_options
- with_quality_scores
- with_completenesses